openapi-cli/src/main/java/io/ballerina/openapi/cmd/BallerinaCodeGenerator.java

```diff
@@ -97,9 +98,8 @@ public class BallerinaCodeGenerator {
      * Generated source will be written to a ballerina module at {@code outPath}
      * <p>Method can be user for generating Ballerina mock services and clients</p>
      */
-    public void generateClientAndService(String definitionPath, String serviceName,
-                                         String outPath, Filter filter, boolean nullable,
-                                         boolean isResource, boolean generateServiceType,
+    public void generateClientAndService(String definitionPath, String serviceName, String outPath, Filter filter,
```

Comment:
Shall we have one data store model for parsing all the parameter values?

Reply:
There is no parameter change done here. Just a reformat.